src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Interaction/Occurrence/Occurrence.vue

```diff
@@ -102,7 +102,7 @@ export default {
       > .message
   ) {
   bottom: -17px; /* Move the absolutely positioned return message to the bottom. -17 to offset the padding of Occurrence. */
-  height: 0;
+  /* height: 0; */
```

Comment:
This rule is removed as it makes the style-panel overlaps the editable return message due to the message container height being zero.